packages/plugin-core/src/constants.ts

```diff
@@ -814,6 +814,11 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     },
     when: "dendron:pluginActive",
   },
+  SHOW_RELEASE_NOTES: {
+    key: "dendron.showReleaseNotes",
+    title: `${CMD_PREFIX} Show Release Notes`,
+    when: "dendron:pluginActive",
```

Comment:
can we show the release notes without the plugin being active? i'm thinking of all the lapsed users that might not have a dendron ws open (or even an active user that isn't in a dendron workspace when reloading vscode)

Reply:
Unfortunately I don't think this is possible based on the way I coded this up, because I need an engine to create the webview contents